website/pages/docs/enterprise/audit-logging.mdx

```diff
@@ -0,0 +1,19 @@
+---
+layout: docs
+page_title: Consul Enterprise Audit Logging
+sidebar_title: Audit Logging
+description: >-
+  Consul Enterprise provides the ability to write events of user behavior with Consul’s API so operations and security users can perform legal compliance auditing.
+---
+
+# Audit Logging
+
+<EnterpriseAlert>
+  This feature is available in all versions of <a href="https://www.hashicorp.com/products/consul/">Consul Enterprise</a>.
```

Comment:
Shouldn't this mention 1.8+?

Reply:
Yes!

I see in the other Enterprise feature pages, in the EnterpriseAlert area, it's written that the feature is available in a particular module. Ex. Namespaces requires the Policy and Governance module (https://www.consul.io/docs/enterprise/namespaces). I see from the pricing page (https://www.hashicorp.com/products/consul/pricing/), Audit Logging falls under the same category. To stay consistent, I can reword to: 'This feature requires Consul Enterprise with the Governance and Policy module.'

I can also add 'Consul Enterprise v1.8.0 adds Audit Logging...' to the first paragraph.